schemas/document-generation-completed.go

```diff
@@ -0,0 +1,18 @@
+package schemas
+
+//  DocumentGenerationCompleted struct for document-generation-completed topic
```

Comment:
Please use only a single space after the double slash.

Reply:
Remove in [ce005fb](https://github.com/companieshouse/kafka-management-tool/pull/1/commits/ce005fb8926cbc2899d61cf66de233a4e80ce97c)